src/lang/Messages.properties

```diff
@@ -2407,7 +2407,9 @@ view.options.label.editorView               = Request editor view style:
 view.options.label.exampleText				= Example text (default size)
 view.options.label.fontExample				= Font Example:
 view.options.label.fontName					= Font Name:
-view.options.label.fontSize					= Font Size:
+view.options.label.fontSize         		= Font Size:
+view.options.label.httpMessageFontName		= HTTP Message Font Name:
```

Comment:
This should be more generic, probably something like "Request/Response Panels Font Name" (the text area is used by other message types e.g. WebSocket) or "Work Panels Font Name" if this is going to be used by other work panels (e.g. Script Console).